// polaris-react/src/server/config.js
/**
 * COPYRIGHT JOSH MUIR 2018
 * UNAUTHORISED ACCESS TO THIS FILE CONSTITUTES AN OFFENSE.
 * YOU WILL BE BLACKLISTED FROM THE SERVICE AND LEGAL ACTION MAY BE TAKEN.
 * Queries - files@muir.xyz
 */


if (process.env.NODE_ENV === "production") {
	// Production values
	module.exports = {
		port: 80,
		voteSecret: "QsNaDtxcjCu3uPSD25Dt",
		voteWebhook: "https://discordapp.com/api/webhooks/527503068288516106/NKrn9gCVbNpJnb8MfrCGK9oOJfbZDE3X3vvYL_RBEv5STXx7YWcuGzPgKZCCNjyB_Stk",
		CLIENT_ID: '375408313724043278',
		CLIENT_SECRET: 'sT6S2ukqzE4BfhBoliGDRpZc6cXUfiEk',
		botInvite: "https://discordapp.com/oauth2/authorize?client_id=375408313724043278&scope=bot&permissions=470281408",
		baseurl: "https://polaris-bot.xyz",
		sentryToken: "https://bdd30ece10c84acd8a131604669b84b1@sentry.io/242368",
		ratelimitHook: "https://discordapp.com/api/webhooks/527505967483715604/bf6aX2YGojH7kmsYeR6o2K5p2m956HYeCoAJsbfrO3M5cowc2qEoxSjxTFqibhBvEVsd"
	};
} else {
	// dev values - test bot
	module.exports = {
		port: 81,
		voteSecret: "QsNaDtxcjCu3uPSD25Dt",
		voteWebhook: "https://discordapp.com/api/webhooks/527503068288516106/NKrn9gCVbNpJnb8MfrCGK9oOJfbZDE3X3vvYL_RBEv5STXx7YWcuGzPgKZCCNjyB_Stk",
		CLIENT_ID: '314776952546525186',
		CLIENT_SECRET: 'hojKseWb4ZVCIHXA0Lric8VB37ce9tfr',
		botInvite: "https://discordapp.com/oauth2/authorize?client_id=314776952546525186&scope=bot&permissions=470281408", //test bot
		baseurl: "http://localhost",
		panelPort: 3000,
		sentryToken: "https://bdd30ece10c84acd8a131604669b84b1@sentry.io/242368",
		ratelimitHook: "https://discordapp.com/api/webhooks/527505967483715604/bf6aX2YGojH7kmsYeR6o2K5p2m956HYeCoAJsbfrO3M5cowc2qEoxSjxTFqibhBvEVsd"
	};
	console.log(`In dev mode.`);
}
